apps/site/pages/courses/[courseSlug]/index.tsx

```diff
@@ -191,6 +192,8 @@ function CourseHeader({
 	summary: CourseProps["summary"];
 	content: CourseProps["content"];
 }) {
+	const session = useSession({ required: false });
```

Comment:
hier ist der `AuthorizedGuard`-Guard bzw die `Unauthorized`-Komponenten vermutlich sinnvoll. 

Vielleicht sogar den AuthorGuard?

Reply:
`useAuthentication` in `@self-learning/ui/layouts`implementiert, da dort auch die anderen Guards liegen.
Der neue Hook ermöglicht:
- Laden der (modifizierten) Seite. Existierende Guards verhindern Laden der kompletten Seite, das ist hier aber nicht erwünscht
- Seite wird geladen, Links zum Lerninhalt werden deaktiviert
- Anstelle zum Kurs zu springen, wird zum Login weiter geleitet. Entsprechend wird auch ein anderer Text angezeigt
- Links zu den Nanomodulen in der Kursübersicht sind deaktiviert